jaxws-addressing/client/pom.xml

```diff
@@ -44,6 +44,11 @@
             <artifactId>jaxws-addressing-service</artifactId>
             <classifier>classes</classifier>
         </dependency>
+        <!-- The JAXWS Client BOM provides management dependencies compatible with the app server,
+        useful for building, testing or debugging JAXWS client applications. Import it to the app's pom.xml
+        dependencyManagement and then set dependencies on specific artifacts, or alternatively simply set
+        a single dependency on it, for the app to depend on all the artifacts it manages, as done here.
+        -->
```

Comment:
useful for--> used for
 Import it to the app's pom.xml-->  Import the BOM to the app's pom.xml
dependencyManagement: Is it a section? Can we say "Import the BOM to the dependency management section of the app's pom.xml file"?
The second sentence is too long. I would split it into two: 
"Import the BOM to the dependency management section of the app's pom.xml file and then set dependencies on specific artifacts. Alternatively, as shown in the following example, set a single dependency on the BOM for the app to depend on all the artifacts it manages."
What do you think?

Reply:
@emmartins: The suggestions above are applicable for content in other sections too.

Reply:
@emmartins, I agree with the rewording suggested by @snelluli1. 
Besides, I would also change the first sentence from:
"The JAXWS Client BOM provides management dependencies compatible with the app server,"
to 
"The JAXWS Client BOM provides Wildfly application server compatible dependencies management,"
WDYT?

Reply:
@snellulli1 will change all considering your suggestions, thanks
@fabiobrz I would prefer to have no references to the app server "name", since that requires changes of content when applied on EAP builds, so I am thinking on just "The JAXWS Client BOM provides application server compatible dependency management...", is that ok for you?

Reply:
@emmartins yes, ok. I agree on stripping the application server name out of the sentence.

Reply:
@emmartins, @snelluli1, just a second thought on the above mentioned last sentence. 
Maybe "compliant" would be more appropriate than "compatible"? Just checking here, WDYT?

Reply:
@fabiobrz: I would say "compatible" is a better word choice. "Compliance" has more to do with legal matters than software systems.

Reply:
@emmartins, @fabiobrz: Just a minor suggestion. looks good to me otherwise. Thanks!